scss/_variables.scss

```
@@ -1696,6 +1696,9 @@ $modal-header-padding-y:            0 !default;
 $modal-header-padding-x:            $spacer !default;
 $modal-header-padding:              $modal-header-padding-y $modal-header-padding-x !default; // Keep this for backwards compatibility
 
+$modal-body-margin-bottom:          $spacer * .5 !default; // Boosted mod
+$modal-body-margin-sm-bottom:       $spacer !default; // Boosted mod
```

Shouldn't the value be equal to `$spacer * .75` ?

---

I'm not 100% sure about these changes since we can't really access it on specs but I think it would better fit this way !
```suggestion
$modal-body-margin-sm-bottom:       $spacer * .75 !default; // Boosted mod
```